src/main/java/uk/ac/ebi/subs/fileupload/errors/SubmissionNotExistsException.java

```diff
@@ -0,0 +1,10 @@
+package uk.ac.ebi.subs.fileupload.errors;
+
+public class SubmissionNotExistsException extends RuntimeException {
+
+    private static final String SUBMISSION_NOT_EXISTS_MESSAGE = "The submission you tried to query is not exists";
```

Comment:
"_The submission you tried to query does not exist._" or even simpler, just: "_Submission not found._"